# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

int32 SUCCESS=1
int32 TF_ERROR=-1
int32 ROBOT_STATE_ERROR=-2
int32 BASE_NOT_MOVING=-3
int32 error_code

